### DISCUSSION TITLE
Help: Is there a keyboard shortcut to save the current memo please?

### DISCUSSION THREAD

**Discussion by Whyglobaleyes** [usememos/memos#4012] [Q&A] [answered]
Hi All,

I'm a few days into using memos and I love how fast the UX is, thank you.

One thing I'd like to know is how to quickly save a memo please? 
- I imagine something like a `ctrl+s` keyboard shortcut might be suitable. 
- I use a chrome extension called shortkeys which can be helpful, I don't know javascript but if someone can suggest a code - block I can try then I'd be grateful.
thanks in advance

**Comment by RoccoSmit** [comment, marked as answer]
`ctrl` + `enter`

**Reply by Whyglobaleyes** [reply]
Genius, marvellous, thank you. As a blind monkey I guess I might have gotten there eventually. 

<edit: it even works for replies on github comments>
